sdk/formrecognizer/Azure.AI.FormRecognizer/src/ExtractedReceipt.cs

```diff
@@ -179,12 +180,8 @@ private static string ConvertStringValue(string fieldName, IDictionary<string, F
                 // https://github.com/Azure/azure-sdk-for-net/issues/10361
                 dateTimeOffsetValue = value.Type switch
                 {
-                    // TODO: Unsuppress
-                    // https://github.com/Azure/azure-sdk-for-net/issues/10376
-#pragma warning disable CA1305 // Specify IFormatProvider
-                    FieldValueType.Date => DateTimeOffset.Parse(value.ValueDate),
-                    FieldValueType.Time => DateTimeOffset.Parse(value.ValueTime),
-#pragma warning restore CA1305 // Specify IFormatProvider
+                    FieldValueType.Date => DateTimeOffset.Parse(value.ValueDate, CultureInfo.InvariantCulture),
+                    FieldValueType.Time => DateTimeOffset.Parse(value.ValueTime, CultureInfo.InvariantCulture),
```

Comment:
It seems the values returned from the service are in the format "yyyy-MM-dd" for dates and "HH:mm:ss" for time. Should we consider calling `ParseExact` here?

Reply:
Can you provide an example of how you would call it?

Reply:
Something like this:
```
FieldValueType.Date => DateTimeOffset.ParseExact(value.ValueDate, "yyyy-MM-dd", CultureInfo.InvariantCulture),
FieldValueType.Time => DateTimeOffset.ParseExact(value.ValueTime, "HH:mm:ss", CultureInfo.InvariantCulture),
```
It will throw a `FormatException` if the values are not in the expected format. The current code is able to parse the info correctly, though. My major concern was in case the service changed the format and, instead of throwing, we ended up parsing it incorrectly (switching days and months, for instance).

However, I think I'm just being overconcerned. We'll have unit tests capable of identifying an issue like this in case it happens.

Reply:
Great, thanks for the example!  Yes, let's use ParseExact.

Reply:
Just for the record: upon further internal discussion we decided to keep `Parse`. `ParseExact` would require a new change every time the service changes the format, while `Parse` should be able to handle changes in general. Parsing issues will be detected by the tests we will have.